label_studio/projects/api.py

```diff
@@ -397,6 +399,31 @@ def get(self, *args, **kwargs):
         return super(ProjectSummaryAPI, self).get(*args, **kwargs)
 
 
+class ProjectSummaryResetCreatedLabelsAPI(GetParentObjectMixin, generics.CreateAPIView):
```

Comment:
Please add a unit test for this API. Also, is the `IsAuthenticated` permission not needed here? I do see that `GetParentObjectMixin` will check object permissions, but I'm not sure if that means we don't need to check authentication. Either way, this could be checked in a test.

Reply:
there is below:
```
  permission_required = ViewClassPermission(
        POST=all_permissions.projects_change,
    )
```
isn't it enough?

Reply:
tests are added

Reply:
https://github.com/HumanSignal/label-studio/pull/5432/files#diff-fc2a039da0975ee919925c1f66e53984db5044b15d741eed50f66837eae03364R141-R148 <- this test specifically covers what happens when a logged out user hits this endpoint